machine_management/control_plane_machine_management/cpmso-using.adoc

```diff
@@ -117,4 +118,12 @@ include::modules/machineset-gcp-shielded-vms.adoc[leveloffset=+2]
 ** link:https://cloud.google.com/compute/shielded-vm/docs/shielded-vm#integrity-monitoring[Integrity monitoring]
 
 //Enabling customer-managed encryption keys for a machine set
-include::modules/machineset-gcp-enabling-customer-managed-encryption.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/machineset-gcp-enabling-customer-managed-encryption.adoc[leveloffset=+2]
+
+[id="cpmso-supported-features-openstack_{context}"]
+== Enabling {rh-openstack} features for control plane machines
```

Comment:
Re: https://github.com/openshift/openshift-docs/pull/64454#discussion_r1357073836

I don't have any problem with diverging from the pattern here to make the statement accurate. How do folks feels about this:
```suggestion
== Updating the configuration for {rh-openstack} control plane machines
```

Reply:
Better than mentioning it as enabling a feature.